### PR TITLE
Add Correctness Test Labeled Event proto.

### PR DESCRIPTION
**Pull request by yunyeng**
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43)

### CODE REVIEW

**Review by wangyaopw — CHANGES_REQUESTED** [review]
I don't think we need such details in the correctnessTest.

The big query or whatever query would generate a list of VID for each EDP. The input to the correctness test would be these VID lists. The query language doesn't need the proto definition I think.

**Review by kungfucraig — COMMENTED** [review]
This is the schema for the big query table. When we parse the CSV we can use the proto to ensure that the values in the column conform to the set of enum values in the schema. This will prevent problems.

Maybe this is not the right place to put this?

> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43#-:-MZnVdHACH--Y6g2Ws7a:b-xv94xb)* status: 0 of 2 files reviewed, all discussions resolved

File: src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    INCOMPLETE = 0;
+    COMPLETE = 1;
+  }
+  Complete complete = 6;
+
+  // Virtual ID of the person.
+  int64 vid = 7;
+
+  // Date of the impression.
+  google.type.Date date = 8;
```
Comment (wangyaopw) [inline src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto:73]: does the sheet have this column?

**Review by wangyaopw — APPROVED** [review]

File: src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto (hunk 1 above)
Comment (yunyeng) [reply src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto:73]: Yes

File: src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+// Input VID data for sketch generation within Correctness Test.
+message LabeledEvent {
+  // Identifier of the publisher.
+  int64 publisher_id = 1;
+
+  // Identifier of the event.
+  int64 event_id = 2;
+
+  // Type of sex.
+  enum Sex {
```
Comment (wangyaopw) [inline src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto:33]: {sex, age, social grad} is only used inside the sheet to get the query parameters. The data in the sheet only contains the integer demographics. In the query we will also use the integer values. So it seems that there is no need to parse the integers back to {sex, age, social grad}

Comment (wangyaopw) [reply src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto:33]: nvw, I see the updated sheet now.

**Review by yunyeng — COMMENTED** [review]
Directory should be ok. There is already another proto in the same place that CorrectnessTest uses for output.

> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43#-:-MZxipE75M_ayEfN6Qip:b-sujhb6)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @wangyaopw and @yunyeng)

File: src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto (hunk 2 above)
Comment (kungfucraig) [inline src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto:33]: Please use SEX_UNSPECIFIED = 0 and then add the other two.

File: src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    MALE = 0;
+    FEMALE = 1;
+  }
+  // Sex of the person.
+  Sex gender = 3;
```
Comment (kungfucraig) [inline src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto:38]: Either "Sex sex" or rename enum to Gender.

**Review by kungfucraig — CHANGES_REQUESTED** [review]

**Review by yunyeng — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43)* status: 0 of 2 files reviewed, 6 unresolved discussions (waiting on @kungfucraig and @wangyaopw)

___
*[src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto, line 33 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43#-MZckQ40lbAtYekt3Mw2-r1-33:-MZxqjmC7qxV7lDBHamB:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/47944b6a902ae86b66c29d12d19c558931da81cd/src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto#L33)):*
<details><summary><i>Previously, kungfucraig (Craig Wright) wrote…</i></summary><blockquote>

Please use SEX_UNSPECIFIED = 0 and then add the other two.
</blockquote></details>

Done.
___
*[src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto, line 38 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43#-MZckQ40lbAtYekt3Mw2-r1-38:-MZxqmnA7yR-ixORdqjY:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/47944b6a902ae86b66c29d12d19c558931da81cd/src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto#L38)):*
<details><summary><i>Previously, kungfucraig (Craig Wright) wrote…</i></summary><blockquote>

Either "Sex sex" or rename enum to Gender.
</blockquote></details>

Done.
___
*[src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto, line 42 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43#-MZckQ40lbAtYekt3Mw2-r1-42:-MZxqnVy2c5igmfF607M:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/47944b6a902ae86b66c29d12d19c558931da81cd/src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto#L42)):*
<details><summary><i>Previously, kungfucraig (Craig Wright) wrote…</i></summary><blockquote>

Same here. AGE_GROUP_UNSPECIFIED = 0
</blockquote></details>

Done.
___
*[src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto, line 56 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43#-MZckQ40lbAtYekt3Mw2-r1-56:-MZxqnsl4YEN4MnSYbMA:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/47944b6a902ae86b66c29d12d19c558931da81cd/src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto#L56)):*
<details><summary><i>Previously, kungfucraig (Craig Wright) wrote…</i></summary><blockquote>

Same SOCIAL_GRADE_UNSPECIFIED
</blockquote></details>

Done.
___
*[src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto, line 64 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43#-MZckQ40lbAtYekt3Mw2-r1-64:-MZxqoFg-xHBOua1FRu3:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/47944b6a902ae86b66c29d12d19c558931da81cd/src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto#L64)):*
<details><summary><i>Previously, kungfucraig (Craig Wright) wrote…</i></summary><blockquote>

Same
</blockquote></details>

Done.
___
*[src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto, line 73 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/43#-MZckQ40lbAtYekt3Mw2-r1-73:-MZxqosO04KtjZFMyQTG:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/47944b6a902ae86b66c29d12d19c558931da81cd/src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto#L73)):*
<details><summary><i>Previously, yunyeng (Yunus Yen) wrote…</i></summary><blockquote>

Yes
</blockquote></details>

Done.

**Review by kungfucraig — APPROVED** [review]

**Pull request merged**